### ISSUE TITLE
The required file html/chunk.xsl from docbook-xsl was not found on the

### ISSUE THREAD

**Issue by Elimelech** [bibletime/bibletime#256]
How to extend the assembly of the program? Where to change options Cmake?

```
root@F3ja:/home/luba # cd /home/luba/bibletime/
root@F3ja:/home/luba/bibletime # sh build-release.sh

....

CMake Error at cmake/BTDocumentation.cmake:224 (MESSAGE):
  The required file html/chunk.xsl from docbook-xsl was not found on the
  system! Please specify its full path with
  -DBT_DOCBOOK_XSL_HTML_CHUNK_XSL=path/to/html/chunk.xsl
Call Stack (most recent call first):
  CMakeLists.txt:212 (INCLUDE)


-- Configuring incomplete, errors occurred!
See also "/home/luba/bibletime/build/CMakeFiles/CMakeOutput.log".
See also "/home/luba/bibletime/build/CMakeFiles/CMakeError.log".
```

**Comment by jaakristioja** [comment]
Do you have docbook-xsl installed?

**Comment by jaakristioja** [comment]
Are you trying to build BibleTime as the root user? What operating system (and version) are you using?

**Comment by Elimelech** [comment]
docbook-xsl installed
I am using freeBSD 12.1
And what needs to be collected on behalf of the user, is it really possible?


2020-07-19 21:18 GMT, Jaak Ristioja <notifications@github.com>:
> Are you trying to build BibleTime as the root user? What operating system
> (and version) are you using?
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/bibletime/bibletime/issues/256#issuecomment-660711349


-- 
Желаю вам поддержки Свыше и успеха!

С уважением,
Ваш  Владимир Тимофиев <http://copi.ru/94717/>
тел./viber: +380504948657

**Comment by jaakristioja** [comment]
If you're building from git, please pull and try again. Or apply commit ece42cfc419e019d8ffc6b690e8e5641b061b438 manually to the source code.

This might help.

If it doesn't you will need to figure out where docbook-xsl installed the `html/chunk.xsl` file and build BibleTime without using CMake directly (or by modifying the respective `build-*.sh` script so that `cmake` gets passed the correct `-DBT_DOCBOOK_XSL_HTML_CHUNK_XSL=path/to/html/chunk.xsl` option).

**Comment by Elimelech** [comment]
I put together with this setup:

cmake -DBT_DOCBOOK_XSL_HTML_CHUNK_XSL=/usr/local/share/xsl/docbook/html/chunk.xsl

2020-07-19 22:52 GMT, Jaak Ristioja <notifications@github.com>:
> If you're building from git, please pull and try again. Or apply commit
> ece42cfc419e019d8ffc6b690e8e5641b061b438 manually to the source code.
>
> This might help.
>
> If it doesn't you will need to figure out where docbook-xsl installed the
> `html/chunk.xsl` file and build BibleTime without using CMake directly (or
> by modifying the respective `build-*.sh` script so that `cmake` gets passed
> the correct `-DBT_DOCBOOK_XSL_HTML_CHUNK_XSL=path/to/html/chunk.xsl`
> option).
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/bibletime/bibletime/issues/256#issuecomment-660721421


-- 
Желаю вам поддержки Свыше и успеха!

С уважением,
Ваш  Владимир Тимофиев <http://copi.ru/94717/>
тел./viber: +380504948657

**Comment by jaakristioja** [comment]
Ok. :smiley: Closing.

**Comment by Elimelech** [comment]
In file included from
/home/luba/bibletime/src/backend/keys/../drivers/cswordmoduleinfo.h:19:
/home/luba/bibletime/src/backend/keys/../drivers/../managers/clanguagemgr.h:166:28:
warning:
      'split' is deprecated: Use Qt::SplitBehavior variant instead
[-Wdeprecated-declarations]
            return abbrevs.split( ";", QString::KeepEmptyParts,
Qt::CaseSensitive );
                           ^
/usr/local/include/qt5/QtCore/qstring.h:603:23: note: 'split' has been
explicitly marked deprecated
      here
    Q_REQUIRED_RESULT QT_DEPRECATED_VERSION_X_5_15("Use
Qt::SplitBehavior variant instead")
                      ^
/usr/local/include/qt5/QtCore/qglobal.h:372:45: note: expanded from
macro
      'QT_DEPRECATED_VERSION_X_5_15'
# define QT_DEPRECATED_VERSION_X_5_15(text) QT_DEPRECATED_X(text)
                                            ^
/usr/local/include/qt5/QtCore/qglobal.h:294:33: note: expanded from
macro 'QT_DEPRECATED_X'
#  define QT_DEPRECATED_X(text) Q_DECL_DEPRECATED_X(text)
                                ^
/usr/local/include/qt5/QtCore/qcompilerdetection.h:675:55: note:
expanded from macro
      'Q_DECL_DEPRECATED_X'
#    define Q_DECL_DEPRECATED_X(text) __attribute__ ((__deprecated__(text)))
                                                      ^
1 warning generated.
[  4%] Building CXX object
CMakeFiles/bibletime_backend.dir/src/backend/keys/cswordldkey.cpp.o
^Z


2020-07-20 12:45 GMT, Jaak Ristioja <notifications@github.com>:
> Closed #256.
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/bibletime/bibletime/issues/256#event-3565625513


-- 
Желаю вам поддержки Свыше и успеха!

С уважением,
Ваш  Владимир Тимофиев <http://copi.ru/94717/>
тел./viber: +380504948657

**Comment by jaakristioja** [comment]
@Elimelech: ??? The build output you pasted also seems to be quite irrelevant to this issue.
